packages/vue/src/examples/pages/login/Login.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <div id="sign-in">
+    <SfModal :visible="true">
+      <transition name="fade" mode="out-in">
+        <div v-if="isLogin" key="log-in" class="form">
+          <SfInput
+            v-model="email"
+            name="email"
+            label="Your email"
+            class="form__input"
+          />
+          <SfInput
+            v-model="password"
+            name="password"
+            label="Password"
+            type="password"
+            class="form__input"
+          />
+          <SfCheckbox
+            v-model="rememberMe"
+            name="remember-me"
+            label="Remember me"
+            class="form__checkbox"
+          />
+          <SfButton class="sf-button--full-width">Login</SfButton>
+          <div class="form__message">
+            <SfButton class="sf-button--text button--muted"
+              >Forgotten password?</SfButton
+            >
+          </div>
+          <div class="form__aside">
+            Don't have and account yet?
+            <SfButton class="sf-button--text" @click="isLogin = false"
+              >Register today?</SfButton
+            >
+          </div>
+        </div>
+        <div v-else key="sign-up" class="form">
+          <SfInput
+            v-model="email"
+            name="email"
+            label="Your email"
+            class="form__input"
+          />
+          <SfInput
+            v-model="firstName"
+            name="first-name"
+            label="First Name"
+            class="form__input"
+          />
+          <SfInput
+            v-model="lastName"
+            name="last-name"
+            label="Last Name"
+            class="form__input"
+          />
+          <SfInput
+            v-model="password"
+            name="password"
+            label="Password"
+            type="password"
+            class="form__input"
+          />
+          <SfCheckbox
+            v-model="createAccount"
+            name="create-account"
+            label="I want to create an account"
+            class="form__checkbox"
+          />
+          <SfButton class="sf-button--full-width submit"
+            >Create an account</SfButton
+          >
+          <div class="form__message">
+            or
+            <SfButton class="sf-button--text" @click="isLogin = true"
```

Comment:
i think we should ship some styles for links like this by default

Reply:
But, it's not a link is a button. We use this style many time, mostly for "actions" not to change url :)